src/modelize/modelize_property.nit

```diff
@@ -880,6 +966,14 @@ redef class AMethPropdef
 		var mysignature = self.mpropdef.msignature
 		if mysignature == null then return # Error thus skiped
 
+		# Check
+		if nsig != null then
+			if not nsig.check_signature(modelbuilder, mclassdef) then
+				self.mpropdef.msignature = null # invalidate
```

Comment:
no, the idea it that the signature is wrong so I invalidate it.
No signature is better than a wrong one. The error was already generated by the `check_signature` call. The null is just to avoid that a wrong signature survives the validation method (and cause pain and despair later)